app/src/main/java/com/x8bit/bitwarden/ui/platform/components/content/BitwardenContentBlock.kt

```diff
@@ -55,29 +65,46 @@ private fun BitwardenContentBlock(
     headerText: AnnotatedString,
     modifier: Modifier = Modifier,
     headerTextStyle: TextStyle = BitwardenTheme.typography.titleSmall,
-    subtitleText: String? = null,
+    subtitleText: AnnotatedString? = null,
     subtitleTextStyle: TextStyle = BitwardenTheme.typography.bodyMedium,
+    showDivider: Boolean = true,
     @DrawableRes iconVectorResource: Int? = null,
     backgroundColor: Color = BitwardenTheme.colorScheme.background.secondary,
 ) {
+    var dividerStartPadding by remember { mutableStateOf(0.dp) }
+    val localDensity = LocalDensity.current
+
     Row(
         modifier = modifier
             .fillMaxWidth()
-            .background(backgroundColor),
+            .background(backgroundColor)
+            .bottomDivider(
```

Comment:
Do we have an scenarios where the incoming `modifier` will have a padding applied?

If so we might want to do something like this:
```kotlin
       modifier = Modifier
            .fillMaxWidth()
            .background(backgroundColor)
            .bottomDivider(
                enabled = showDivider,
                paddingStart = dividerStartPadding,
            )
            .then(modifier),
```

That will enforce that the background and divider is applied before any padding on the modifier.